types/formidable/Formidable.d.ts

```diff
@@ -12,17 +18,17 @@ declare class IncomingForm {
      */
     parse(request: IncomingMessage, callback: (err: any, fields: Fields, files: Files) => void): void;
 
-    once(name: "end", callback: () => void): void;
-    once(name: "error", callback: (err: any) => void): void;
+    once(eventName: 'end', listener: () => void): this;
+    once(eventName: 'error', listener: (err: any) => void): this;
```

Comment:
these methods return `this` based on the extended class

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/events.d.ts#L359